docs/index.asciidoc

```diff
@@ -103,6 +103,123 @@ Common causes are:
  - When the hit result contains top-level fields that are {logstash-ref}/processing.html#reserved-fields[reserved in Logstash] but do not have the expected shape. Use the <<plugins-{type}s-{plugin}-target>> directive to avoid conflicts with the top-level namespace.
  - When <<plugins-{type}s-{plugin}-docinfo>> is enabled and the docinfo fields cannot be merged into the hit result. Combine <<plugins-{type}s-{plugin}-target>> and <<plugins-{type}s-{plugin}-docinfo_target>> to avoid conflict.
 
+[id="plugins-{type}s-{plugin}-cursor"]
+==== Tracking a field's value across runs
+
+NOTE: experimental:[] `tracking_field` and related settings are experimental and subject to change in the future
+
```

Comment:
```suggestion
.Technical Preview: Tracking a field's value
****
The feature that allows tracking a field's value across runs is in _Technical Preview_.
Configuration options and implementation details are subject to change in minor releases without being preceded by deprecation warnings.
****

```

Reply:
We don't need both notes, so please decide which treatment and working you like.